tealiumlibrary/src/main/java/com/tealium/core/collection/DeviceCollector.kt

```diff
@@ -63,9 +64,11 @@ class DeviceCollector private constructor(private val context: Context) : Collec
         windowManager.defaultDisplay.getSize(it)
         "${it.x}x${it.y}"
     }
-    override val deviceLogicalResolution = point.let {
-        windowManager.defaultDisplay.getRealSize(it)
-        "${it.x}x${it.y}"
+    override val deviceLogicalResolution = DisplayMetrics().let { metrics ->
+        windowManager.defaultDisplay.getRealMetrics(metrics)
+        val x = (metrics.widthPixels / metrics.density).roundToInt()
```

Comment:
this way of rounding seems to be a pixel out for some devices - checked against the expected results from the card.
Suggest that we use `ceil` and `toInt` instead to round them up as they seem to match whats listed elsewhere as well as the examples on the card:
```
val x = ceil(metrics.widthPixels / metrics.density).toInt()
val y = ceil(metrics.heightPixels / metrics.density).toInt()
```